controllers/index.js

```diff
@@ -1,6 +1,16 @@
 const router = require('express').Router();
 const homeRoutes = require('./homeRoutes');
+const adminRoutes = require('./adminRoutes');
+const eventRoutes = require('./eventRoutes');
+const galleryRoutes = require('./galleryRoutes');
+const contactRoutes = require('./contactRoutes');
+const aboutRoutes = require('./aboutRoutes');
 
 router.use('/', homeRoutes);
+router.use('/admin', adminRoutes);
+router.use('events', eventRoutes);
```

Comment:
Should be `'/events'`. Route is currently broken and returns a 404 error when it should be a 500. Just put the / and it should fix it :)

Reply:
That is so cool that you can highlight a certain line or lines and add a comment 😂 
Thanks Tony, it's fixed now! 👍